docs/0___preamble.adoc

```diff
@@ -44,6 +44,11 @@ _History / Road Map_
 |Bugfix/maintenance release of FMI for Model Exchange and Co-Simulation 2.0
 |====
 
+|2.0.4	
+|2022-09-13	
+|Bugfix/maintenance release of FMI for Model Exchange and Co-Simulation 2.0; Backporting of “orthogonal” features “Terminals and Icons” and “Build Configuration” from FMI 3.0
```

Comment:
Not sure if you want to use proper double quotes here and in the other occurrences.

Reply:
For FMI 2.0.x the adoc is for tracking only, the PDF will be created from the Word document.